### ISSUE TITLE
Visual bug when using texture packs

### ISSUE THREAD

**Issue by Kynz0** [SpyMan10/backpackmod-1.16-old#32]
When i use texture pack that changes gui of inventory it makes this weird visual bug.
I use vanilla tweaks dark UI.
![image](https://user-images.githubusercontent.com/90714597/194298431-19b42702-4638-4b29-bfad-be62301eccc2.png)
![image](https://user-images.githubusercontent.com/90714597/194298488-81b23875-d9b0-4551-bf9d-26e775f3aebe.png)

**Comment by SpyMan10** [comment]
Hello, It's not a bug, GUI textures are generated automatically because size of each backpack item is configurable

**Comment by Kynz0** [comment]
ok but is there a way to fix it?

**Comment by SpyMan10** [comment]
No, the only way to fix that is to add a new property in the configuration file to specify the color to use to fill the gui background. I will fix in the next major release (mc 1.20). That mod need a complete rewrite, the current code is fully deprecated.

But if you want you can recompile it with the new background color.
File :
https://github.com/SpyMan10/backpackmod/blob/252dad85f0784dad5d3b8b9c98f28197cc8a7936/src/main/java/net/spyman/backpackmod/client/gui/screen/BackpackScreen.java#L58
